docs/aerodromes/classc/Sydney.md

```diff
@@ -283,59 +283,115 @@ Climb Gradient Requirements apply to all Procedural SIDs. It is the pilot's resp
 | Between 2000FT & 3000FT| >5000M         | `EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL`|
 | Below 2000FT **or**    | <5000M         | `EXP INSTR APCH`                |
 
-When Independent Visual Approaches are run during PROPS, the APCH field must also include:  
-`DO NOT PASS THRU ASSIGNED RWY CL`
+!!! Important
+    Independent Visual Approaches (IVAs) may only be run during Parallel Runway Operations (PROPS). During single runway operations, regular visual approaches (blank ATIS 'APCH' field) should be used instead.
+
+When Independent Visual Approaches are run during PROPS, the APCH field must also include `DO NOT PASS THRU ASSIGNED RWY CL`.
 
 ### Operational Info
 
-When parallel runways are used for departures but inbound traffic levels don't necessitate independent approaches (low traffic levels), the ATIS OPR INFO shall include:  
-`INDEP PARL DEPS IN PROG`  
+> For the following approach types:
+> 
+> - `EXP INSTR APCH`
+> 
+> - `EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL`
+> 
+> the ATIS OPR INFO shall include `INDEP PARL DEPS IN PROG`.
+> 
+> !!! Note
+    This permits independent departures (two simultaneous departures on parallel runways) but NOT independent approaches (two simultaneous arrivals on parallel runways without perscribed separation minima).
+
+⠀  
+
+> For the following approach type,
+> 
+> - `EXP INDEP VISUAL APCH`
+> 
+> the ATIS OPR INFO shall include `INDEP PARL APPROACHES AND DEPS IN PROG`.
+
+⠀  
+
+> When weather conditions are very poor (such as during thunderstorms), the use of independent approaches and departures are prohibited and the ATIS OPR INFO shall include `PARL RWY OPS IN PROGRESS`.
+> 
+> !!! Note
+    This means that Sydney ADC cannot depart two aircraft simultaneously on parallel runways.
```

Comment:
What seperation is required?

Reply:
Visual sep or 3nm. If visual sep cannot be assured until the departure paths are separated by 3nm, then the departures must be staggered, ie, an aircraft cannot depart from the Western runway until a departure from the eastern runway is 3nm clear of the departure path.

Which is just another way this introduces unnecessary complications in my opinion. I would prefer to keep it simple, which works for 99.99% of vatsim operations